defaults/main.yml

```diff
@@ -2,8 +2,10 @@
 # defaults file for template
 storage_backend: "default"
 pool_layers: ["pool-partitions", "vg"]  # md, luks, vdo under vg
+tests_list: ['verify-volumes']
```

Comment:
You'll want to pay attention to the variable names since basically all variables in ansible are global AIUI.